frontend/src/concepts/pipelines/kfTypes.ts

```diff
@@ -108,7 +108,7 @@ export type PipelineSpecKF = {
   pipeline_name?: string;
   workflow_manifest?: string;
   pipeline_manifests?: string;
-  parameters: ParameterKF[];
+  parameters?: ParameterKF[];
```

Comment:
Is there a need to change the base type?

Reply:
ahh , I had the same discussion in one of the other pr , We need to be mindful about the types ,  so in theory we shouldn't optional access an attribute we know is mandatory, so I have changed the types

Reply:
Ok i think making this optional is ok. I found this doc which mentions `parameters` as optional.
https://github.com/kubeflow/pipelines/blob/master/backend/api/v1beta1/python_http_client/docs/ApiPipelineSpec.md

Reply:
oh yes @christianvogt in the race we had for feature completion we might have slipped some types, so is great to review some of the types from time to time to verify them, since we might have mistyped some of them.
Is not that usual but it could have happened.